public/locales/en/translation.json

```
@@ -157,7 +157,7 @@
     "enterCounterPartyAddressOrENS": "Enter counterparty address or ENS",
     "copyLink": "Copy link",
     "expiryShouldBeMoreThan0": "Expiry should be more than 0",
-    "enterValidTakerAddress": "Enter valid taker address",
+    "enterValidTakerAddress": "Enter address",
```

```suggestion
    "enterValidTakerAddress": "Enter valid taker address",
```

Not part of this ticket.

---

it's now part of https://github.com/airswap/airswap-web/issues/679